### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Resistantsmile** [jacobfortune/Songs-Of-War-Addon#5] [Q&A] [answered]
I can't import the mcaddon on my phone can you help me?

**Comment by lesleyespire** [comment]
Hello, what might be the problem? I'll help however I can.
________________________________
From: Resistantsmile ***@***.***>
Sent: 15 January 2024 15:56
To: ThetruePyrobyte/Songs-Of-War-Addon ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ThetruePyrobyte/Songs-Of-War-Addon] Help (Discussion #5)


I can't import the mcaddon on my phone can you help me?

**Comment by lesleyespire** [comment]
The mcaddon file should download fine, as far as I know, though I have seen issues with phones
________________________________

**Comment by lesleyespire** [comment, marked as answer]
For anyone else seeing this, the problem is solved by getting a copy of the addon on a mcworld. If you need one, just contact me, this is only for people struggling to download the mcaddon file.

If not, another way to fix this issue is to download the addon as a zip and then convert it to a mcaddon, the stable version of the addon is at https://mcpedl.com/songs-of-war-addon/. BlazeDrake is the main pioneer but I am the current developer.
